bpmsuite/bpmsuite70-businesscentral-monitoring-ephemeral.json

```diff
@@ -0,0 +1,506 @@
+            "value": "openshift",
+            "required": true
+        },
+        {
+            "displayName": "EAP Admin User",
+            "description": "EAP administrator username",
+            "name": "ADMIN_USERNAME",
+            "value": "eapadmin",
+            "required": false
+        },
+        {
+            "displayName": "EAP Admin Password",
+            "description": "EAP administrator password",
+            "name": "ADMIN_PASSWORD",
+            "from": "[a-zA-Z]{6}[0-9]{1}!",
+            "generate": "expression",
+            "required": false
+        },
+        {
+            "displayName": "KIE Admin User",
+            "description": "KIE administrator username",
+            "name": "KIE_ADMIN_USER",
+            "value": "adminUser",
+            "required": false
+        },
+        {
+            "displayName": "KIE Admin Password",
+            "description": "KIE administrator password",
+            "name": "KIE_ADMIN_PWD",
+            "from": "[a-zA-Z]{6}[0-9]{1}!",
+            "generate": "expression",
+            "required": false
+        },
+        {
+            "displayName": "KIE Server Controller User",
+            "description": "KIE server controller username (adds the user)",
+            "name": "KIE_SERVER_CONTROLLER_USER",
+            "value": "controllerUser",
+            "required": false
+        },
+        {
+            "displayName": "KIE Server Controller Password",
+            "description": "KIE server controller password (adds the user)",
+            "name": "KIE_SERVER_CONTROLLER_PWD",
+            "value": "controllerPassword",
+            "required": false
+        },
+        {
+            "displayName": "KIE Server User",
+            "description": "KIE execution server username (Sets the org.kie.server.user system property, used by the embedded controller)",
+            "name": "KIE_SERVER_USER",
+            "value": "executionUser",
+            "required": false
+        },
+        {
+            "displayName": "KIE Server Password",
+            "description": "KIE execution server password (Sets the org.kie.server.pwd system property, used by the embedded controller)",
+            "name": "KIE_SERVER_PWD",
+            "value": "executionPassword",
+            "required": false
+        },
+        {
+            "displayName": "Smart Router listening port",
+            "description": "Port in which the smart router server listens (router property org.kie.server.router.port)",
+            "name": "KIE_ROUTER_PORT",
+            "value": "9000",
+            "required": false
+        },
+        {
+            "displayName": "Smart Router external URL",
+            "description": "Public URL where the router can be found. Format http://<host>:<port> (router property org.kie.server.router.url.external)",
+            "name": "KIE_ROUTER_URL_EXTERNAL",
+            "example": "http://my-app-smartrouter-projectName.osurl:9000",
```

Comment:
Here it would be great if it could be possible to automatically set this variable to the external route of the smart router